src/main/java/uk/gov/digital/ho/hocs/domain/repositories/EnumMappingsRepository.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.digital.ho.hocs.domain.repositories;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.domain.repositories.entities.EnumMappings;
+
+import java.util.List;
+
+@Slf4j
+@Service
+public class EnumMappingsRepository extends JsonConfigFileReader {
```

Comment:
OOS for this - but what if we want different mappings based on the type of message. Maybe one to write a ticket for?

Reply:
Do you mean that we might want to map enums to a different value based on complaint type?

Reply:
Yeah very much the same as https://github.com/UKHomeOffice/hocs-casework/tree/main/src/main/resources/config/summary but for message type. 

This can be a new ticket though...